### PR TITLE
added tests to shard utils ShardIDs

### PR DESCRIPTION
**Pull request by IakimovNutria**

### CODE REVIEW

File: pkg/shard/mock_shard.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import (
+	"crypto/tls"
+	"github.com/jackc/pgx/v5/pgproto3"
+	"github.com/pg-sharding/spqr/pkg/config"
+	"github.com/pg-sharding/spqr/pkg/conn"
+	"github.com/pg-sharding/spqr/pkg/models/kr"
+	"github.com/pg-sharding/spqr/pkg/txstatus"
+)
+
+type mockShard struct {
```
Comment (EinKrebs) [inline pkg/shard/mock_shard.go:12]: please use `MockShard` from `mock` package

**Review by EinKrebs — CHANGES_REQUESTED** [review]

File: pkg/shard/mock_shard.go (hunk 1 above)
Comment (IakimovNutria) [reply pkg/shard/mock_shard.go:12]: I can't because it causes import cycle

Comment (reshke) [reply pkg/shard/mock_shard.go:12]: anyway you should use mockgen.
What is the import cycle are you struggling with?

Comment (IakimovNutria) [reply pkg/shard/mock_shard.go:12]: > What is the import cycle are you struggling with?

package github.com/pg-sharding/spqr/pkg/shard
	imports github.com/pg-sharding/spqr/pkg/mock/shard
	imports github.com/pg-sharding/spqr/pkg/shard: import cycle not allowed in test

Comment (EinKrebs) [reply pkg/shard/mock_shard.go:12]: Consider changing package name in your test to shard_test

**Review by EinKrebs — APPROVED** [review]

**Pull request merged**